owners/index.js

```diff
@@ -42,15 +43,19 @@ module.exports = app => {
    * @param {!PullRequest} pr pull request to run owners check on.
    */
   async function runOwnersCheck(github, pr) {
-    const ownersCheck = new OwnersCheck(github, pr);
     let checkRunId;
     let latestCheckRun;
 
     try {
-      const approvers = await ownersCheck.getApprovers();
+      const localRepo = new LocalRepository(process.env.GITHUB_REPO_DIR);
+      const ownersCheck = new OwnersCheck(localRepo, github, pr);
       checkRunId = await github.getCheckRunId(pr.headSha);
-      const fileOwners = await Owner.getOwners(github, pr.number);
-      latestCheckRun = ownersCheck.buildCheckRun(fileOwners, approvers);
+
+      // TODO: Once OwnersCheck swallows the remaining logic in Owner, this can
```

Comment:
Always write TODO(#issuenumber) or TODO(username) if there's no issue (issue number is preferable)

Reply:
Done